core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreConstants.java

```diff
@@ -245,6 +245,8 @@
         setAdvancedProperty(JDBCCaseInsensitiveConstants.ADD_USER_PROPERTY_CASE_INSENSITIVE_OPENEDGE, "Add User " +
                 "Property With Case Insensitive Username (OpenEdge)", JDBCCaseInsensitiveConstants
                 .ADD_USER_PROPERTY_OPENEDGE_SQL_CASE_INSENSITIVE, "");
+        setAdvancedProperty(UserStoreConfigConstants.claimOperationsSupported, "Claim Operations Supported", "true",
```

Comment:
can we make "Claim Operations Supported" also a constant as it is used in several places? just a suggestion.